cmake/onnxruntime_optimizer.cmake

```diff
@@ -30,8 +30,12 @@ if (onnxruntime_MINIMAL_BUILD)
 
   if (onnxruntime_ENABLE_RUNTIME_OPTIMIZATION_IN_MINIMAL_BUILD)
     list(APPEND onnxruntime_optimizer_src_patterns
+      "${ONNXRUNTIME_ROOT}/core/optimizer/conv_activation_fusion.h"
+      "${ONNXRUNTIME_ROOT}/core/optimizer/conv_activation_fusion.cc"
       "${ONNXRUNTIME_INCLUDE_DIR}/core/optimizer/graph_transformer_utils.h"
       "${ONNXRUNTIME_ROOT}/core/optimizer/graph_transformer_utils.cc"
+      "${ONNXRUNTIME_ROOT}/core/optimizer/initializer.h"
+      "${ONNXRUNTIME_ROOT}/core/optimizer/initializer.cc"
```

Comment:
What the binary size cost of pulling in all the code in Initializer? Do the unused parts get dropped by the linker so we don't need to ifdef that out?

Reply:
![image](https://user-images.githubusercontent.com/18449977/156069627-acfce12b-49e0-4301-9783-e7cdaaebb29c.png)
3.9kb from initializer.cc
I didn't see unused Initializer methods in SizeBench so I think they are excluded.

Reply:
Initializer can probably be refactored to use tensorprotoutils helpers more. could do that in a separate change.